elements/pl-file-editor/pl-file-editor.js

```diff
@@ -95,10 +101,14 @@ window.PLFileEditor.prototype.syncSettings = function () {
     if (event.key === 'pl-file-editor-theme') {
       this.editor.setTheme(event.newValue);
     }
+    if (event.key === 'pl-file-editor-textsize') {
+      this.editor.setFontSize(event.newValue);
+    }
   });
 
   window.addEventListener('pl-file-editor-settings-changed', () => {
     this.editor.setTheme(localStorage.getItem('pl-file-editor-theme'));
+    this.editor.setFontSize(localStorage.getItem('pl-file-editor-textsize'));
```

Comment:
Whenever retrieving values from local storage, you'll need to take care to convert them back into an integer, as you do in the constructor. The fact that that's not currently being done is causing sync across multiple editors to fail. You'll need to make the same change in the `storage` event listener above.

Alternatively, you can store the value as a pixel string (e.g. `16px`). That might be a cleaner option so we're not constantly converting back and forwards between options.